### DISCUSSION TITLE
Unable to change device bay status.

### DISCUSSION THREAD

**Discussion by tzrafi** [netbox-community/netbox#17204] [Help Wanted!]
I need to setup a device bay for a server. I have created the device bay but can't change its status from vacant.
![image](https://github.com/user-attachments/assets/a30fde0e-c961-48ee-853b-a6a69a38b5bf)
![image](https://github.com/user-attachments/assets/ab47b3ef-d8d7-4795-b5ee-e3d26b8a816c)

**Comment by candlerb** [comment]
Have you tried installing a child device in the device bay?
